### PR TITLE
add TVOS support and update native android and ios agents

### PR DESCRIPTION
**Pull request by ndesai-newrelic**

### CODE REVIEW

File: Plugins/NewRelic/Source/NewRelic/ThirdParty/IOS/NewRelic.xcframework/Info.plist [hunk 1]
```diff
@@ -8,32 +8,32 @@
 			<key>DebugSymbolsPath</key>
 			<string>dSYMs</string>
 			<key>LibraryIdentifier</key>
-			<string>ios-arm64_x86_64-maccatalyst</string>
```
Comment (cdillard-NewRelic) [inline Plugins/NewRelic/Source/NewRelic/ThirdParty/IOS/NewRelic.xcframework/Info.plist:11]: Did this change come with the new framework or did you manually edit this?

Comment (ndesai-newrelic) [reply Plugins/NewRelic/Source/NewRelic/ThirdParty/IOS/NewRelic.xcframework/Info.plist:11]: it came with the new framework

Comment (cdillard-NewRelic) [reply Plugins/NewRelic/Source/NewRelic/ThirdParty/IOS/NewRelic.xcframework/Info.plist:11]: excellent we should be golden then

File: Config/DefaultEngine.ini [hunk 2]
```diff
@@ -220,8 +220,8 @@ bUseManualIPAddress=False
 ManualIPAddress=
 
 [/Script/IOSRuntimeSettings.IOSRuntimeSettings]
-MobileProvision=fc364b1f-31f5-43e1-8104-8985132f4e6f.mobileprovision
-SigningCertificate=Apple Development: Nisarg Desai (GF42H6D58V)
+MobileProvision=58712ff3-db48-40b0-ad2c-f08f85eb2557.mobileprovision
+SigningCertificate=iPhone Distribution: New Relic Inc (SU7SUNGZJP)
```
Comment (cdillard-NewRelic) [inline Config/DefaultEngine.ini:224]: Why is the New Relic team distribution required in this Repo?

Comment (ndesai-newrelic) [reply Config/DefaultEngine.ini:224]: i think i need to delete this file.

**Review by cdillard-NewRelic — APPROVED** [review]

**Pull request merged**